### ISSUE TITLE
Current Folder to favorites

### ISSUE THREAD

**Issue by ERGPOL** [files-community/Files#9514]
### What's the Problem?

want to pin network devices to favorites.  Network devices are not listed under network drivers. 
there are no other ways to do it

### Solution/Idea

Current Folder to favorites

### Alternatives

--

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |

### Windows Version

Edition	Windows 11 Pro Version	21H2 Installiert am	‎05.‎06.‎2021 Betriebssystembuild	22000.778 Leistung	Windows Feature Experience Pack 1000.22000.778.0

**Comment by yaichenbaum** [comment]
You can right click in the network location to add it to the favorites section.

**Comment by ERGPOL** [comment]
That’s right, but only when devices are listen there. In my case they are not listed

**Comment by yaichenbaum** [comment]
You should be able to right click in the drive itself in a blank area, you don't have to do it from the server.

**Comment by ERGPOL** [comment]
Ok, will try it

**Comment by ERGPOL** [comment]
it works. thank you
